py/picca/delta_extraction/data.py

```diff
@@ -213,6 +213,13 @@ def filter_forests(self):
                 self.logger.progress(
                     f"Rejected forest with los_id {forest.los_id} "
                     "due to finding nan")
+            elif self.analysis_type=='PK 1D' and forest.mean_snr<1:
```

Comment:
Maybe I'm wrong but it does not seem that putting a space in a str is a good idea. Why not just 'PK1D'?

Reply:
this is currently just how it's defined in delta_extraction, not sure at how many places this would need renaming. Also for BAO it is defined as BAO 3d, so it's currently always <statistics> <space> <dimensionality>. I don't really mind doing it either way, but consistency would be most important here...